### ISSUE TITLE
Unable to access previously purchased content 

### ISSUE THREAD

**Issue by vsobris** [hymns-for-worship/issues#108]
I am unable to access content I’ve previously purchased (sheet music and slides packs for all collections). This has been the case since the update in January. This has occurred previously with past updates, but would eventually be corrected. Logging out and back in does not fix the issue. Restoring purchases does not fix the issue. I have been unable to view any sheet music and am instead prompted to make another purchase.

**Comment by rodydavis** [comment]
Please test in version 4.0.0 of the app!

**Comment by vsobris** [comment]
I’ll keep my eyes out for its availability in the iOS App Store! I’ll let you know my results. Thank you.

Kind regards,

Vince Sobris

**Comment by rodydavis** [comment]
Should be live now!

**Comment by vsobris** [comment]
Thank you! I am once again able to access my purchased content, so that’s great!

I see there is another overhaul to the layout of the app. I have a couple questions.

Is there a way to access the “favorites” list of songs from the previous app version?

And…

Is there a way to stay zoomed in on sheet music? On iOS, when I pinch to zoom, it will zoom out after a couple of seconds.

Kind regards,

Vince Sobris

**Comment by rodydavis** [comment]
You got it!

For the UI overhaul I focused a lot on making the core experience of the app great and eliminating the rest or pushing it to a static website.

For favorites I would suggest making a playlist for that. Everything is built around playlists now and optimized for it.

For the persistent zoom I can look into it, maybe can make it a setting in the future! I have it go to the default zoom to help with users that accidentally zoom and not know what to do to get back.

**Comment by vsobris** [comment]
It seems to be working.

Only issues I’ve noticed are:

1) Searching for a song by song number (ID?) yields several results. Usually the desired song is second in the list of results. This occurs even when the ID filter is selected as the only filter. I did have one instance where the desired song did not show in the results until I restarted the app.

2) The “Add Hymn” button within a playlist does not work after creating a new playlist until you exit and re-open the playlist.

3) When adding hymns to the playlist using the “Add Hymn” button it looks like we can select multiple hymns to add at once since you used checkboxes, but only one can be selected at a time. Not really a bug, but being able to select multiple would make for better user experience.

4) A date is required when creating a playlist. Again, maybe not a bug.

Other than that, is there a way to view all of the various hymnals and collections you had in the previous version of the app?

Thank you!

Kind regards,

Vince Sobris
